website/integrations/services/argocd/index.md

```diff
@@ -21,44 +21,30 @@ The following placeholders are used in this guide:
 :::
 
-## authentik Configuration
+## authentik configuration
 
-### Step 1 - Provider creation
+To support the integration of ArgoCD with authentik, you need to create an application/provider pair in authentik.
 
-In authentik, create an _OAuth2/OpenID Provider_ (under _Applications/Providers_) with these settings:
+### Create an application and provider in authentik
 
-- Name: ArgoCD
-- Client Type: `Confidential`
-- Signing Key: Select any available key
-- Redirect URIs:
+1. Log in to authentik as an admin, and open the authentik Admin interface.
+2. Navigate to **Applications** > **Applications** and click **Create with Provider** to create an application and provider pair. (Alternatively you can create only an application, without a provider, by clicking **Create**.)
 
-```
-https://argocd.company/api/dex/callback
-http://localhost:8085/auth/callback
-```
-
-After creating the provider, take note of the `Client ID` and `Client Secret`, you'll need to give them to ArgoCD in the _ArgoCD Configuration_ field.
-
-### Step 2 - Application creation
-
-Create a new _Application_ (under _Applications/Applications_) with these settings:
-
-- Name: ArgoCD
-- Provider: ArgoCD
-- Slug: argocd
-- Launch URL: https://argocd.company/auth/login
-
-### Step 3 - ArgoCD Group creation
+- **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
+- **Choose a Provider type**: select **OAuth2/OpenID Connect** as the provider type.
+- **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.
+    - Note the **Client ID**,**Client Secret**, and **slug** values because they will be required later.
+    - Add two `Strict` redirect URI and set them to <kbd>https://<em>argocd.company</em>/api/dex/callback/</kbd> and <kbd>https://<em>localhost:8085</em>/auth/callback/</kbd>.
+    - Select any available signing key.
+- **Configure Bindings** _(optional)_: you can create a [binding](/docs/add-secure-apps/flows-stages/bindings/) (policy, group, or user) to manage the listing and access to applications on a user's **My applications** page.
 
-Create a new _Group_ (under _Directory/Groups_) that'll be used as the admin group for ArgoCD (if you already have an "admin" group, you can skip this part!)
+3. Click **Submit** to save the new application and provider.
 
-- Name: ArgoCD Admins
-- Members: Add your user and/or any user that should be an ArgoCD admin
+### Create the users and administrator groups
 
-You can create another group for read-only access to ArgoCD as well if desired:
+Using the authentik Admin interface, navigate to **Directory** -> **Groups** and click **Create**. ArgoCD lets you to set up administrator users and read-only users by creating groups named `ArgoCD Admins` and `ArgoCD Viewers`.
```

Comment:
Odd phrasing on "ArgoCD lets you to set up..."

Also, let's not use the phrase "let's you" here if these are required groups. We can be more prescriptive/forceful.